arch/risc-v/src/common/supervisor/riscv_sbi.c

```diff
@@ -33,24 +45,30 @@
  * Private Functions
  ****************************************************************************/
 
-#ifdef CONFIG_ARCH_RV64
-static inline uint64_t rdtime(void)
+static inline uintptr_t sbi_ecall(unsigned int extid, unsigned int fid,
```

Comment:
should we move the general sbi_ecall to riscv_internal.h?

Reply:
Sure it can be done. Do you anticipate the definition will be needed by other source files ? My original idea was to put all SBI calls in this file but I don't know if e.g. OpenAMP or some IPI-interface etc. will need this ?

Reply:
Ok, let's merge this version first until we found the need.